docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -346,11 +346,11 @@ If you intend to scan Illumina runs, you will have to build `runscanner-illumina
 
 Start Tomcat on this machine.
 
-Edit `$CATALINA_HOME/conf/Catalina/localhost/miso.properties` and set `miso.runscanner.urls` to the URL of the Run Scanner instance and restart MISO.
+Edit `$CATALINA_HOME/conf/Catalina/localhost/miso.properties` of the MISO Tomcat server and set `miso.runscanner.urls` to the URL of the Run Scanner instance. Restart MISO.
 
 It is possible to set up multiple run scanners managing different sequencers and add all the URLs to `miso.properties`.
 
-You can view the run scanner's state from the main page of the Run Scanner server.
+You can view the run scanner's state from the main page of the Run Scanner server: `http://<run-scanner-URL>:8080`.
```

Comment:
I think URL/port wasn't mentioned here because it depends on Tomcat/server config, which is outside the scope of these instructions